### PR TITLE
Update openoffice.rb

### PR DESCRIPTION
**Pull request by ybhwang**
Korean added

After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/openoffice.rb [hunk 1]
```diff
@@ -25,10 +25,16 @@
...
+  
+  language 'ko' do
+    sha256 '2c9798bde42c6d758b2a189aa5dd1170bdfed9c6346fdf8de43a276ee1de2c80'
+    'ko'
+  end
 
   # sourceforge.net/openofficeorg.mirror was verified as official when first introduced to the cask
   url "https://downloads.sourceforge.net/openofficeorg.mirror/Apache_OpenOffice_#{version}_MacOS_x86-64_install_#{language}.dmg"
   appcast 'https://sourceforge.net/projects/openofficeorg.mirror/rss'
+  # appcast 'https://sourceforge.net/mirror/openofficeorg/activity/feed'
```
Comment (reitermarkus) [inline Casks/openoffice.rb]: is this the same as the one above? How does it differ?

Comment (YBHwang) [reply Casks/openoffice.rb]: First I added my country's language Korean.
Second I need to receive an answer in the Issues. (commented)

Comment (reitermarkus) [reply Casks/openoffice.rb]: Is `https://sourceforge.net/projects/openofficeorg.mirror/rss` not working, or why do you want to add `https://sourceforge.net/mirror/openofficeorg/activity/feed`?

`https://sourceforge.net/mirror/openofficeorg/activity/feed` does not even contain the version `4.1.6`.

Comment (YBHwang) [reply Casks/openoffice.rb]: I saw it contains other Languages more than before.. Just It!

**Pull request merged**